### PR TITLE
Create expose_alertmanager.sh

### PR DESCRIPTION
**Pull request by zeari**
Enables you to access the alertmanager ui and api

### CODE REVIEW

File: expose_alertmanager.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    app: prometheus
+  sessionAffinity: None
+  type: ClusterIP
+status:
+  loadBalancer: {}
+EOF
+  
+  
+oc expose service alertmanager
+route="`oc get routes -n openshift-metrics alertmanager | grep alertmanager | awk  '{print $2}'`"
```
Comment (elad661) [inline expose_alertmanager.sh]: maybe use `$()` instead of backticks (more noticable and thus more readable)

File: expose_alertmanager.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+cat <<EOF | oc create -n openshift-metrics -f -
```
Comment (elad661) [inline expose_alertmanager.sh:3]: add shabang `#!/bin/bash`

**Review by elad661 — COMMENTED** [review]
two minor nitpicks, otherwise looks good.

**Comment by moolitayer** [comment]
cc @zgalor

**Pull request merged**